model_analyzer/config/generate/search_parameters.py

```diff
@@ -49,11 +49,13 @@ def __init__(
 
         self._populate_search_parameters()
 
-    def get_parameters(self) -> List[SearchParameter]:
-        return [v for v in self._search_parameters.values()]
+    def get_parameter(self, name: str) -> Optional[SearchParameter]:
+        try:
```

Comment:
Doesn't dict.get() perform this for you?

Reply:
It does (apparently I like re-inventing the wheel!). Fixed.